qemu/tests/hotplug_unplug_during_io_repeat.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import time
+
+from virttest import error_context
+from virttest import utils_misc
+from virttest import utils_disk
+
+from provider.block_devices_plug import BlockDevicesPlug
+from provider.storage_benchmark import generate_instance
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test hotplug of block devices.
+
+    1) Boot up guest with virtio-blk or virtio-scsi system disk.
+    2) Hoplug a virtio-blk or virtio-scsi data disk by qmp.
+    3) Do read/write data on hotplug block.
+    4) Unplug block device during serving block io, then verify devices.
+    5) repeat step2~step4 300 times.
+
+    :param test:   QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env:    Dictionary with test environment.
+    """
+    def _check_iozone_status():
+        ck_session = vm.wait_for_login(timeout=360)
+        if not utils_misc.wait_for(
+                lambda: check_cmd[os_type].split()[-1] in ck_session.cmd_output(
+                    check_cmd[os_type]), 180, step=3.0):
```

Comment:
it's wired that I tested found the fact waiting time is 1800 secs:
23:05:08 ERROR| ERROR 1-Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.hotplug_unplug_during_io_repeat.q35 -> TestError: Not all IOZONE.EXE processes done in 1800 sec.

Reply:
Hit the same error

Reply:
Hi, @kimi1978wy  @peixiu  I have updated the codes, pls help me to review it, thanks.
CC @XueqiangWei @zhenyzha